src/olympia/constants/search.py

```diff
@@ -21,7 +21,7 @@
     'danish': ['da'],
     'german': ['de'],
     'greek': ['el'],
-    'english': ['en-us'],
+    'english': ['en-us', 'en-gb'],
```

Comment:
the extracted strings are always extracted into a list so it'll look like that:

```
{
    'name_l10n_english': [ 'Banana Bacon (US)', 'Banana Bacon (GB)'']
    'name_l10n_spanish': ['Banana Bacon Espanol']
}
```

I wonder if we should make a special case for english to only index either but we do exactly the same for Chinese as well and I think it someone wants to have an explicit en-GB locale we should index it.